common/app/views/main.scala.html

```diff
@@ -80,7 +80,7 @@
 
         @fragments.inlineJSNonBlocking()
 
-        @fragments.analytics(page)
+        @fragments.analytics.analytics(page)
```

Comment:
@sndrs I have done something like that. The other file did not only contain Omniture, so I broke it up a bit.